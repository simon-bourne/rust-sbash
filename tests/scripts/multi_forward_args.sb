pub fn test-fn(arg0, arg1, $@) {
    echo "$@"
}
