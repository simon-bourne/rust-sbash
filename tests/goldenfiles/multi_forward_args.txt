test-fn () { ( arg0="$1"; shift; arg1="$1"; shift; 
    echo "$@"
) };